docs/src/contributing/development-environment.md

```diff
@@ -1,280 +1,266 @@
 
 Homebrew users can just install with cross compiling support:
 
-
-    $ brew install go --with-cc-common
+```
+$ brew install go --with-cc-common
+```
 
 It is also straightforward to build Go from source:
 
-    $ sudo su
-    $ curl -sSL https://golang.org/dl/go1.5.src.tar.gz | tar -v -C /usr/local -xz
-    $ cd /usr/local/go/src
-    $ # compile Go for our default platform first, then add cross-compile support
-    $ ./make.bash --no-clean
-    $ GOOS=linux GOARCH=amd64 ./make.bash --no-clean
+```
+$ sudo su
+$ curl -sSL https://golang.org/dl/go1.5.src.tar.gz | tar -v -C /usr/local -xz
+$ cd /usr/local/go/src
+$ # compile Go for our default platform first, then add cross-compile support
+$ ./make.bash --no-clean
+$ GOOS=linux GOARCH=amd64 ./make.bash --no-clean
+```
 
-Once you can compile to `linux/amd64`, you should be able to compile Deis' components as normal.
+Once you can compile to `linux/amd64`, you should be able to compile Deis components as normal.
 
 ### Configuring Docker Machine (Mac)
 
-Deis needs a Docker registry running independently of the Deis cluster. On OS X, you will need [Docker Machine][machine] to run the registry inside of a VirtualBox image.
-
-!!! note
-    Previously, Deis used boot2docker to run the registry. However, Docker has deprecated boot2docker in favor of Docker Machine.
+Deis needs Docker for building images.  Docker utilizes a client/server architecture, and while the Docker client is available for Mac OS, the Docker server is dependent upon the Linux kernel.  Therefore, in order to use Docker on Mac OS, [Docker Machine][machine] is used to facilitate running the Docker server within a [VirtualBox][vbox] VM.
 
-Install Docker Machine according to the normal installation instructions. Then create a new image for hosting your Deis Docker registry:
+Install Docker Machine according to the normal installation instructions, then use it to create a new VM:
 
-    $ docker-machine create --driver virtualbox --virtualbox-disk-size=100000 \
-      --engine-insecure-registry=192.168.0.0/16 deis-registry
+```
+$ docker-machine create deis-docker \
+    --driver=virtualbox \
+    --virtualbox-disk-size=100000 \
+    --engine-insecure-registry=192.168.0.0/16
+```
 
-This will create a new virtual machine named `deis-registry` that will take up as much as 100,000 MB of disk space. Registries tend to be large, so allocating a big disk is a good idea.
+This will create a new virtual machine named `deis-docker` that will take up as much as 100,000 MB of disk space. The images you build may be large, so allocating a big disk is a good idea.
 
-Once the deis-registry machine exists, source its values into your environment so your docker client knows how to use the new machine.
+Once the `deis-docker` machine exists, source its values into your environment so your docker client knows how to use the new machine. You may even choose to add this to your bash profile or similar.
```

Comment:
sometimes on my machine, the machine comes up but the docker daemon takes 30-60 seconds more to become responsive. your call if you want to mention it.